tests/unit/dojson/test_dojson_hep_bd2xx.py

```diff
@@ -136,3 +136,37 @@ def test_titles_from_245__a_b():
     result = hep2marc.do(result)
 
     assert expected == result['245']
+
+
+def test_titles_from_245_and_247__a_9():
```

Comment:
we only had 6 records in HEP with 247 content, which I all changed to 247, so this test can be removed.